### ISSUE TITLE
Tab indentation issue in injectGlobal

### ISSUE THREAD

**Issue by yangaobo** [styled-components/stylelint-processor-styled-components#160]
`injectGlobal` indentation rules work differently with `Tab`
I copied your test case https://github.com/styled-components/stylelint-processor-styled-components/blob/master/test/fixtures/inject-global/valid.js and update the indentation from 2-space to Tab
```js
import {injectGlobal} from 'styled-components';

injectGlobal`
	html {
		color: blue;
	}
`;
```
and config my stylelint as follow
```
{
  "processors": ["stylelint-processor-styled-components"],
  "extends": [
    "stylelint-config-standard",
    "stylelint-config-styled-components"
  ],
  "syntax": "scss",
  "rules": {
    "indentation": "tab"
  }
}
```
stylelint fails with
```
 4:2  ✖  Expected indentation of 0 tabs   indentation
 5:3  ✖  Expected indentation of 1 tab    indentation
 6:2  ✖  Expected indentation of 0 tabs   indentation
```
I have to change the code to
```js
injectGlobal`
html {
	color: blue;
}
`;
```
I guess this is a bug? because the rule should behaviour the same for `space` and `tab`?

**Comment by emilgoldsmith** [comment]
Thanks! That's definitely unintended

**Comment by emilgoldsmith** [comment]
The fix is published in 1.2.2 we had a typo in a regex :)
